content-src/components/DiscoveryStreamComponents/DSFooter/_DSFooter.scss

```diff
@@ -0,0 +1,76 @@
+.story-footer {
+  color: var(--newtab-text-secondary-color);
+  display: flex;
+  height: 24px;
+  inset-inline-start: 0;
+  margin-top: 12px;
+  align-items: center;
+  transition: transform 0.3s;
+  transform-style: preserve-3d;
+
+  .story-sponsored-label,
+  .status-message {
+    @include dark-theme-only {
+      color: $grey-40;
+    }
+
+    font-size: 13px;
+    line-height: 24px;
+    color: $grey-50;
+  }
+
+  .story-sponsored-label {
+    color: #737373;
+    transform: translateZ(11px);
+    transition-property: opacity;
+    transition-duration: 200ms;
+  }
```

Comment:
Do we need transition on Sponsored label?

Reply:
Nope. Looks like this is some legacy css and we had a transition there before?